frontend/src/components/pages/DisplayReview/DisplayReview.tsx

```diff
@@ -0,0 +1,87 @@
+import { ArrowBackIcon } from "@chakra-ui/icons";
+import { Box, IconButton, Text } from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router-dom";
+
+import reviewAPIClient from "../../../APIClients/ReviewAPIClient";
+import { Review, ReviewResponse } from "../../../types/ReviewTypes";
+import { mapReviewResponseToReview } from "../../../utils/MappingUtils";
+import LoadingSpinner from "../../common/LoadingSpinner";
+import PreviewReview from "../../PreviewReview/PreviewReview";
+
+/**
+ * The model defining the route parameters that can be passed to
+ * the Display Review Page
+ */
+interface Params {
```

Comment:
Could you name this something specific to this component? (ie `DisplayReviewParams`, etc)